### PR TITLE
fix: [dev-profile] Mobile / Header objects alignment to the left

### PR DESCRIPTION
**Pull request by Adriann77**

### CODE REVIEW

**Review by mr-fox93 — APPROVED** [review]
good job, checked on my phone.

**Review by hansac91 — APPROVED** [review]

File: src/components/Headers/UserProfileHeader/UserProfileHeader.module.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .wrapper {
   position: absolute;
   display: flex;
-  justify-content: flex-end;
```
Comment (mmcode9407) [inline src/components/Headers/UserProfileHeader/UserProfileHeader.module.scss:4]: We style according to the desktop first rule. So move it to the `@include mobile` mixin. The `flex-end` value should stay for divs without `withBackButtonClass` for the desktop.

File: src/components/Headers/UserProfileHeader/UserProfileHeader.module.scss [hunk 2]
```diff
@@ -13,14 +13,20 @@
 }
 
 .withBackBackButton {
-  justify-content: space-between;
```
Comment (mmcode9407) [inline src/components/Headers/UserProfileHeader/UserProfileHeader.module.scss:16]: The same as above. Desktop first, and then add `@include Mobile` with some styles for mobile :)

File: src/components/Headers/UserProfileHeader/UserProfileHeader.module.scss [hunk 3]
```diff
@@ -13,14 +13,20 @@
...
-  align-items: center;
-  justify-content: left;
+  align-items: flex-start;
+  justify-content: flex-start;
   flex-wrap: wrap;
   row-gap: 12px;
   column-gap: 10px;
 }
+
+@media (min-width: 768px) {
```
Comment (mmcode9407) [inline src/components/Headers/UserProfileHeader/UserProfileHeader.module.scss:28]: When you use the `@include Mobile` mixin with `withBackButton` class, you won't need this rule anymore.

**Review by mmcode9407 — CHANGES_REQUESTED** [review]
Looks good, but we have to fix some styles yet :)

**Review by Sumick — APPROVED** [review]

**Review by mmcode9407 — APPROVED** [review]

**Pull request merged**